catroid/src/main/java/org/catrobat/catroid/formulaeditor/Sensors.java

```diff
@@ -40,7 +40,8 @@
 	OBJECT_Y_VELOCITY(true), OBJECT_ANGULAR_VELOCITY(true),
 	LAST_FINGER_INDEX, FINGER_X, FINGER_Y, FINGER_TOUCHED, OBJECT_LOOK_NUMBER(true), OBJECT_LOOK_NAME(true),
 	OBJECT_BACKGROUND_NUMBER(true), OBJECT_BACKGROUND_NAME(true),
-	NFC_TAG_ID(true), NFC_TAG_MESSAGE(true);
+	NFC_TAG_ID(true), NFC_TAG_MESSAGE(true), GAMEPAD_A_PRESSED, GAMEPAD_B_PRESSED, GAMEPAD_UP_PRESSED,
```

Comment:
please put a line break after the NFC fields, it's easier to read if these are blocked by functionality groups